### PR TITLE
[Doc Improvement][Tag mention update for cloud][3351508]

### PR DESCRIPTION
**Pull request by v-preethah**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3f973452476bb2967abd82fb592c05f7c3d47e87/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0a5c94f0-a007-4982-9598-4e016058f06c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cf941da3866cd44b52c58bb9e45e3e69fea2f5f2/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0bfec3c7-9a5d-4fbe-b05c-20153d2b6e9c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ Your bot can mention tags in text messages and Adaptive Cards posted in channels
 >
 > * Tag mentions are available only in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md).
 > * Tag mentions are supported in Teams desktop and web clients. However, it's not supported in Teams mobile client.
-> * Tag mentions are supported in Government Community Cloud (GCC) and GCC-High tenants, and not supported in  Department of Defense (DoD) tenant.
+> * Tag mentions are currently not supported in Government Community Cloud (GCC), GCC-High, and Department of Defense (DoD) tenants.
```
Comment (v-deabira) [inline msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md:308]: Remove currently.
Redraft: Tag mentions aren't supported in Government Community Cloud (GCC), GCC-High, and Department of Defense (DoD) tenants.

**Review by v-deabira — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5159d9d68c7d0313e1161d52adfa2be8336f5d90/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a31a9e2-2336-4e66-9f89-a89af96c0ea8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/14c45f277d1d9bdf277c735e0bf899ed2dfbba87/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c36f87c-d2ba-448c-ad8f-dcacf023011a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md (hunk 1 above)
Comment (v-preethah) [reply msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md]: Updated the change.

**Review by v-deabira — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/23d3c80801665d5aeab7d5dac06afbce2f965b48/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5569fb7d-1de8-496d-bfab-3e8daff5dee9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e0f98475e6eceec0cd7848be0e975e63728a3333/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00b87f6a-1687-4a20-bdc6-8d2c084ae009) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**